### ISSUE TITLE
Blur my shell make crash at startup if enable Ubuntu kinetic

### ISSUE THREAD

**Issue by InitialCrow** [aunetx/blur-my-shell#416]
Hello ty for this ext I use it since long time now.

Yesterday, I redid an installation of my ubuntu kinetic kudu, I made my config and impossible to start gnome, my screens are black with only the mouse displayed.

So I did some research in my system and I concluded that it is the blur shell extension that crashes at startup, the others work. So I redid an installation and before doing my config, I just installed the extension to see if it was not my config that had broken something and the same result.
so the extension crashes the latest version of Ubuntu kinetic kudu but the weird thing is that it was working a week ago on the same config.

Also, if I install the extension and activate it, it works fine, but on restart it crashes gnome.

I think it's from lib-gjs which had an update 3 days ago and I think the extension is preventing it from starting but I'm not at all sure.

**Comment by CorvetteCole** [comment]
I can't seem to reproduce this, set up 22.10 in a virtual machine and have no problems. Any further information you could provide?

**Comment by InitialCrow** [comment]
Hi, sorry, but I can't give more explanation, because I changed os and hardware. So all I can say is how it behaved.
As said above, I was on an Ubuntu with X11 and 4 screens on Nvidia GTX 1070ti and 2 screens on an intel i9 11900k.
The initial problem was that I couldn't get the 6 screens to work under wayland (but that's another subject that has nothing to do with blur my shell). so under and the 6 screens everything worked with blur my shell under ubuntu 22.10 (kinetic), I insist it worked correctly. I tampered with a lot of things to try to make the wayland problem work and I broke the ubuntu more than once. Abandoning the wayland problem (by the way it comes from nvidia which does not manage the multigpu under wayland) I reinstalled on the date of the ticket the ubuntu 22.10 under the same configuration as that which worked and the on the date of the ticket blur my shell crashed ubuntu the only track I could find was the gjs update which could crash because in the mutter logs I had good when the BMS extension was a gjs crash error.

I can no longer send a log because following the problem above I went to amd with an rx7900 xt and for linux kernel concerns I am currently under the beta of ubuntu 23.04 and therefore everything works correctly even under wayland and BMS which just ported to gnome 44 works also i notice it while writing this answer.

So thank you for porting it to gnome 44 early.
and currently on my new config everything works correctly

Small precision on the problem of my old config the crash occurred at the start of gnome if I deactivated the extension, I logged in and I reactivated the extension it worked but would have crashed at the restart. Sorry if I could not be more useful
